File: Mobil.java
// File: Mobil.java
public class Mobil {
    private String merk;
    private int tahun;

    public Mobil(String merk, int tahun) {
        this.merk = merk;
        this.tahun = tahun;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public String getMerk() {
        return merk;
    }

    public void tampilkanInfo() {
        System.out.println("Merk: " + merk + ", Tahun: " + tahun);
    }
}

File: cpp_version/Mobil.cpp
#include "Mobil.h"
#include <iostream>
using namespace std;

// Konstruktor untuk inisialisasi merk dan tahun
Mobil::Mobil(string m, int t) {
    merk = m;
    tahun = t;
}

// Fungsi untuk menampilkan informasi mobil
void Mobil::tampilkanInfo() {
    cout << "Merk Mobil: " << merk << endl;
    cout << "Tahun Pembuatan: " << tahun << endl;
}

// Setter untuk merk
void Mobil::setMerk(string m) {
    merk = m;
}

// Getter untuk merk
string Mobil::getMerk() {
    return merk;
}
